builtin_interfaces/Time stamp           # Time at which it was published
string[] name
float64[] command
int32 mode
int32 POSITION = 0
int32 VELOCITY = 1
int32 TORQUE = 2
int32 UNKNOWN = 3
